internal/cluster/clusteradapter/distribution_eks.go

```diff
@@ -58,3 +58,18 @@ func (s eksService) UpdateNodePool(ctx context.Context, clusterID uint, nodePool
 func (s eksService) DeleteNodePool(ctx context.Context, clusterID uint, name string) (deleted bool, err error) {
 	panic("implement me")
 }
+
+// ListNodePools lists node pools from a cluster.
+func (s eksService) ListNodePools(ctx context.Context, clusterID uint) (nodePoolList cluster.RawNodePoolList, err error) {
+	nodePools, err := s.service.ListNodePools(ctx, clusterID)
+	if err != nil {
+		return nil, errors.WrapWithDetails(err, "listing node pools through EKS service failed", "clusterID", clusterID)
+	}
+
+	response := make(map[string]interface{}, len(nodePools))
```

Comment:
Using a map makes the order inconsistent. Can we avoid it? What's the reason behind using a map here (it's not immediately obvious).

Reply:
Fixed in https://github.com/banzaicloud/pipeline/compare/2ad4cafb067c442a1cfbf1b9cd4071e298b269c8..f640984ef5dee7ae5770cc55c9ccc9741c0e760b.

No reason behind, this was an oversight from me while taking over the feature code.